src/components/paragraphs/stanford-card/card-paragraph.tsx

```diff
@@ -14,6 +14,23 @@ type Props = HtmlHTMLAttributes<HTMLDivElement> & {
   paragraph: ParagraphStanfordCard
 }
 
+const adjustHeadingType = (size: "larger" | "smaller" | "default" | undefined, heading: string): string => {
+  const headingTypes: {[key: string]: number} = {
+    h2: 4,
+    h3: 3,
+    h4: 2,
+  }
+  if (!headingTypes[heading] || !size || size === "default") return ""
+  let type = headingTypes[heading]
+  if (size === "larger") {
+    type += 1
+  } else if (size === "smaller") {
+    type -= 1
+  }
```

Comment:
at this point, we know that the `size` is either `larger` or `smaller`, so we can simplify the condition.

```suggestion
  const type = size === "larger" ? headingTypes[heading] + 1 : headingTypes[heading] - 1
```